### PR TITLE
/posts/ instead of /posts

### PR DESCRIPTION
**Pull request by hlenke**

### CODE REVIEW

File: blog/.vuepress/theme/components/Home.vue [hunk 1]
```diff
@@ -24,7 +24,7 @@
         </div>
 
         <router-link
-          to="/posts"
+          to="/posts/"
```
Comment (maxigimenez) [inline blog/.vuepress/theme/components/Home.vue:27]: Just out of curiosity, why is this needed?

Comment (hlenke) [reply blog/.vuepress/theme/components/Home.vue:27]: The ultimate URL is /posts/ linking to a redirect might mess with our SEO score. And does certainly with our SEO Healthscore on ahrefs.com.

Comment (maxigimenez) [reply blog/.vuepress/theme/components/Home.vue:27]: Interesting, thanks Hannes!

**Review by maxigimenez — APPROVED** [review]
LGTM 👍
